internal-packages/workflow-designer-ui/src/settings/github-integration/github-integration-setting-form.tsx

```diff
@@ -16,6 +19,39 @@ import {
 import { PayloadMapForm } from "./payload-map-form";
 import { useGitHubIntegrationSetting } from "./use-github-integration-setting";
 
+const NEXT_ACTION_DISPLAY_NAMES: Record<
+	WorkspaceGitHubNextIntegrationAction,
+	string
+> = {
+	"github.issue_comment.create": "Create Issue Comment",
+	"github.pull_request_comment.create": "Create Pull Request Comment",
+} as const;
+
+const TRIGGER_TO_ACTIONS: Record<
+	WorkspaceGitHubIntegrationTrigger,
+	WorkspaceGitHubNextIntegrationAction[]
+> = {
+	"github.issues.opened": ["github.issue_comment.create"],
+	"github.issue_comment.created": ["github.issue_comment.create"],
+	"github.pull_request_comment.created": ["github.pull_request_comment.create"],
+	"github.issues.closed": ["github.issue_comment.create"],
+} as const;
+
+const TRIGGERS_REQUIRING_CALLSIGN = [
+	"github.issue_comment.created",
+	"github.pull_request_comment.created",
+] as const;
+
+type TriggerRequiringCallsign = (typeof TRIGGERS_REQUIRING_CALLSIGN)[number];
+
+const requiresCallsign = (
```

Comment:
Suggest renaming `requiresCallsign` to `isTriggerRequiringCallsign` for better clarity and consistency with boolean function naming conventions.


```suggestion
const isTriggerRequiringCallsign = (
```